functionary/ui/views/teams/create.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin, UserPassesTestMixin
+from django.http import HttpRequest, HttpResponse, QueryDict
+from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
+from django.views.decorators.http import require_GET
+from django.views.generic import CreateView
+from rapidfuzz import process
+
+from core.auth import Permission
+from core.models import Team, TeamUserRole, User
+from ui.forms.teams import TeamUserRoleForm
+from ui.views.teams.utils import get_user_from_username
+
+
+class TeamCreateMemberView(LoginRequiredMixin, UserPassesTestMixin, CreateView):
+    model = TeamUserRole
+    form_class = TeamUserRoleForm
+    template_name = "forms/teamuserrole_create_or_update.html"
+
+    def get_context_data(self, **kwargs) -> dict:
+        context = super().get_context_data(**kwargs)
+        team = get_object_or_404(Team, id=self.kwargs.get("team_id"))
+
+        # Use string of id field if it is a UUID
+        context["team_id"] = str(team.id)
+        context["create_user"] = True
+        context["usernames"] = User.objects.all()[:5]
```

Comment:
Let's not do this.  Once there's any meaningful number of users this ends up feeling like a list of 5 random users, none of which is likely to be the one you are looking for.